### PR TITLE
Add easy_thumbnails

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay.

File: recipes/easy-thumbnails/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - django >=1.8
+    - pillow
+
+test:
+  imports:
+    - easy_thumbnails
+
+about:
+  home: https://github.com/SmileyChris/easy-thumbnails
+  license: BASD-3-Clause
```
[the remark this answers is not in the document]
Comment (sannykr) [reply recipes/easy-thumbnails/meta.yaml]: @scopatz : Thanks for pointing out.

File: recipes/easy-thumbnails/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (dougalsutherland) [inline recipes/easy-thumbnails/meta.yaml]: Can you switch to the `pip` line?

Comment (dougalsutherland) [reply recipes/easy-thumbnails/meta.yaml]: Actually I just did it, didn't know you were online. :)

**Pull request merged**